package.json

```diff
@@ -20,7 +20,8 @@
     "build": "tsc",
     "lint": "tslint --project tsconfig.json --format codeFrame 'src/**/*.ts'",
     "fmt": "prettier --write *.{md,js,json} 'src/**/*.ts'",
-    "prepublishOnly": "npm run build"
+    "prepublishOnly": "npm run build",
+    "release": "standard-version --skip.tag"
```

Comment:
Does `standard-version` publish to npm? If so, we should "skip" that part too, since we already have CircleCI setup to handle that for us.

Reply:
From `standard-version`'s readme:

> standard-version takes a different approach by handling versioning, changelog generation, and git tagging for you without automatic pushing (to GitHub) or publishing (to an npm registry). Use of standard-version only affects your local git repo - it doesn't affect remote resources at all.

So we should be safe.

Reply:
Thanks, I probably could/should have looked that up 🤦‍♂